### ISSUE TITLE
5-02-05 Methods for Terrestrial domain - new terms for hydrology

### ISSUE THREAD

**Issue by jbianchi81** [wmo-im/wmds#281]
# **Branch**

https://github.com/wmo-im/wmds/blob/issue281/tables_en/5-02-05.csv

# **Summary and Purpose**

To add terms for methods used in hydrology. Also discuss a new structure for the table according to the suggestion of using tags instead of a path.

# **Stakeholder(s)**

The hydrologic community

# Final Proposal

~ updated 3 June 2022

AMEND the following codes and definitions:

| **Notation** | **Name** | **Description** |
| --- | --- | --- |
| 126 | Gravimetric moisture determination~method~ | Measurement of the amount of water in soil by weighing a sample before and after drying it in an oven at a temperature of 100-110 degrees Celsius. \[Based on ‘gravimetric method’ in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 128 | Soil heat~Heat~ flux plate |   |
| 159 | Neutron probe | Measurement of the moisture content and hence the porosity of a geological formation by means of lowering a tool containing a radioactive element emitting neutrons and a neutron detector into a borehole. \[Based on 'neutron logging' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 187 | Electrical resistance blocks (Porous blocks)~Resistance block tensiometer~ | Determination of electrical resistance between two electrodes embedded in blocks of a porous material buried in the ground and in equilibrium with the surrounding moist soil. \[Based on 'porous blocks/electrical resistance blocks' in Guide to Hydrological Practices (WMO-No. 168), Volume II. 2009 edition.\] |
| 200 | Tensiometer | Measurement of soil capillary tension by means of a porous cup, inserted in the soil, connected to a manometer by a water-filled tube. \[Based on 'tensiometer' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 203 | Time domain reflectometry (TDR) | Estimation of water content in unsaturated materials based on variations in their dielectric properties. \[International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |

Add new codes: 

| **Notation** | **Name** | **Description** |
| --- | --- | --- |
| 345 | Echo sounding | Evaluation of the return time of acoustic waves emitted into a medium to determine the distance between source and target. This method is used in hydrology, for example, to determine the depth of a water body. \[Based on 'echo sounder' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 346 | Frequency domain reflectometry | Measurement of soil capacitance by means of radio frequency waves. The soil acts as the dielectric completing a capacitance circuit, which is part of a feedback loop of a high-frequency transistor oscillator. \[Based on 'Frequency domain reflectometry' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 347 | Gamma-ray attenuation | Determination of gamma-ray attenuation at given depth of soil using a source and a detector placed at a given distance at the same depth. If the apparent dry bulk density of the soil is known the signal can be converted into a measure of the volumetric soil-moisture content. \[Based on 'Gamma-ray attenuation' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 348 | Non-recording gauge | Determination of water stage by means of a non-recording gauge (e.g., graduated vertical or inclined staff gauge, wire-weight gauge, graduated rod, tape or point gauge and maximum-stage gauge). \[Based on 'Non-recording gauges' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 349 | Staff gauge | Determination of water stage by means of a graduated vertical scale, fixed to a staff or structure, on which the water level may be read. Synonym: vertical gauge. \[Based on 'Staff gauge' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 350 | Inclined gauge | Determination of water stage by means of a non-vertical scale which is graduated to indicate vertical heights. \[Based on 'inclined gauge' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 351 | Wire-weight gauge | Determination of water stage by means of a fine cable or wire attached to a weight which is lowered to the water surface in order to determine its position below a fixed above-surface point. \[Based on 'wire weight gauge' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 352 | Point gauge | Level measurement gauge, the essential element of which is a pointed rod that is lowered until it touches the water surface. \[International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 353 | Maximum-stage gauge | Determination of the elevation of the flood crest by the adherence of regranulated cork to a graduated staff held in a fixed position with relation to the datum. \[Based on 'maximum-stage gauge' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 354 | Recording gauge | Determination of water stage by means of a recording gauge, for example, a device that automatically records the water level as detected by a sensor, either continuously or at regular time intervals. Synonyms: limnigraph, stage recorder, water-level recorder. \[Based on 'water-level recorder' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 355 | Wire-float recording gauge (hydrograph) | Determination of water stage by means of a gauge consisting of a float which rises and falls with changes in the water level; this movement being transmitted to a recording or indicating device. \[Based on 'float gauge' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 356 | Pressure-actuated recording gauge | Recording gauge operating on the principle that static pressure at a fixed point in the stream is directly proportional to the head of liquid above the point, i.e. the water level is proportional to the pressure difference between a fixed spot in the water column and the atmospheric pressure. \[Based on 'Recording gauges' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 357 | Velocity-area method | Method of determining the discharge of a stream by measuring the width, velocity, and depth of the flowing water at a number of elemental areas, over the cross-section, and summing the products. \[Based on 'velocity-area method' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 358 | Float gauging | Measurement of the velocity of a stream by means of timing the downstream transit of a float or velocity rod. \[International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 359 | Dilution gauging | Determination of the discharge of a stream by measuring the degree of dilution, caused by the flowing water, of an added tracer solution. Synonyms: chemical gauging, tracer dilution gauging. \[Based on 'dilution gauging' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 360 | Slope-area method | Determination of the discharge in a reach which is based on the surface slope, the reach roughness, the wetted perimeters and the flow areas of the various wetted cross-sections in the reach. \[Based on 'slope-area method' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 361 | Measuring weir | Determination of the discharge of a stream by means of a device or structure across the stream which causes a free or submerged fall. \[Based on 'measuring weir' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 362 | Flume | Determination of the discharge of a stream by means of a man-made channel with clearly specified shape and dimensions. \[Based on 'flume' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 363 | Stage-discharge relation | Determination of the discharge of a stream by means of a relationship between water level and discharge for a given cross-section, which may be expressed as a curve, a table or an equation. \[Based on 'stage-discharge relation' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 364 | Water current meter | Determination of water velocity by means of a current meter, for example, an instrument for measuring water velocity. Synonym: flowmeter. \[Based on 'current meter' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 365 | Cup-type current meter | Determination of water velocity by means of a current meter whose rotor is composed of a wheel fitted with cups turning on a vertical axis. \[Based on 'cup-type current meter' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 366 | Propeller-type current meter | Determination of water velocity by means of a current meter in which the rotor is a propeller rotating around an axis parallel to the flow. \[Based on 'propeller-type current meter' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 367 | Acoustic (ultrasonic) velocity meter | Measurement of the difference in travel time of acoustic (ultrasonic) pulses between transducers in a stream to determine the mean velocity on the signal path. \[Based on 'acoustic (ultrasonic) velocity meter' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 368 | Acoustic doppler current profiler (ADCP) | An instrument that uses an acoustic doppler transducer to measure water velocity, depth and vessel path simultaneously and at various depths to compute discharge. The total discharge is computed from the profile information using matrix multiplication. \[Based on 'Acoustic Doppler Current Profilers' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 369 | Acoustic doppler current meter | Determination of stream velocity by means of an acoustic Doppler instrument, which measures the velocity of the water based on the Doppler shift experienced by an acoustic pulse of energy which is reflected off particles suspended in, and moving with, the water. \[Based on 'Acoustic Doppler Velocimeter' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 370 | Optical or nuclear suspended-sediment gauge | Determination of the decrease of intensity of a visible light or X-rays beam by means of a photoelectric or nuclear detector situated at constant distance from the source of the beam. This decrease, caused by the scattering and/or absorption by suspended-sediment particles, is proportional to the sediment concentration, if other relevant characteristics of water and sediment (chemical, mineral composition, etc.) remain unchanged. \[Based on 'photoelectric or nuclear suspended-sediment gauges' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 371 | Gravimetric sediment measurement | Estimation of the total sediment in a water sample by allowing the sediment to settle in a container, siphoning off the supernatant water, carefully transferring all the sediment from the container on to filter paper, then drying the sample and determining the mass of total sediment. \[Based on 'gravimetric method' in International Glossary of Hydrology (WMO-No. 385). 2012 edition.\] |
| 372 | Displacement method of sediments | Determination of sediment concentration by the measurement of the difference in weight between a sample of sediment-laden water and an equal volume of clear water at the same temperature. \[Based on 'displacement method' in Manual on Sediment Management and Measurement (WMO-No. 948). 2003.\] |
| 373 | Water discharge-sediment discharge rating curve | An empirical relationship relating sediment discharge and streamflow based on correlation analysis of sediment samples and concurrent flow estimates and presented as a curve, table, or equation. \[Based on 'sediment rating curve' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 374 | Water discharge-bed material rating curve | An empirical relationship relating bed-material discharge and streamflow based on correlation analysis of sediment samples and concurrent flow estimates and presented as a curve, table, or equation. \[Based on 'sediment rating curve' in Guide to Hydrological Practices (WMO-No. 168), Volume I. 2008 edition, updated in 2020.\] |
| 375 |  Volumetric method | A process for measuring the discharge of a well, spring, or stream by diverting its flow into a volumetric container and timing the fill. |

# **Proposal Drafts**

## Update 17 Feb 2022

This list was submitted to HCP and shared within JET-HYDMON for feedback from the hydrological community. It contains basic methods used in hydrology. All descriptions are cited from official WMO documents.

## Update 28 Apr 2022

Introducing amendments proposed by @joergklausen [comment](https://github.com/wmo-im/wmds/issues/281#issuecomment-1098772826). Plus some editorial changes to definitions

## Update 18 May 2022

Added notation. Adjusted references format (removed hlinks)

## Update 19 May 2022

Incorporated feedback from HCP/USGS

## Update 1 Jun 2022

Removed formulae and adjusted references syntax

# **Reason**

Provide vocabulary for hydrology

**Comment by joergklausen** [comment]
Thank you @jbianchi81 for this extensive set of proposed methods. Skimming through it, I can see several exist already in the atmosphere table (an obvious example: mercury thermometer). I think, we need to re-consider this split of the tables by domain  to avoid this duplication. 
Further, the methods should refer to physico-chemical principles as much as possible. For example, "BOD test, dilution method": the observed variable is in fact "BOD", so the method should not repeat this if possible. The idea is to be relatively high-level, but it is also true that some procedures are strictly regulated. For this example, I would propose then to be even more specific and refer to it as ISO xyz, US EPA nnn, .... The same is true, for example, for a method like "Winkler" (not Winckler!) which is in fact a "redox titration according to Winkler". So here, like with all the others, my request is to choose a short name ("Winkler titration" or "Winkler test") and provide a description along the lines of ("The Winkler test is used to determine dissolved oxygen in water samples. It involves redox chemistry using manganese and iodide chemicals and acid-base titration. (some reference)"

**Comment by jbianchi81** [comment]
Thanks @joergklausen . I support the idea of unifying the methods tables of all domains. Meanwhile I will delete duplicates and adjust the terms according to your suggestions

**Comment by joergklausen** [comment]
Note after TT-WIGOSMD meeting on 27 May 2021.
Methods are usually not domain-specific, so a split of the methods table according to domain was a bad decision in hind-sight (I think). In the medium term, I suggest completely re-work these tables, add the domains as tags, but only create 1 table. If that is too long, then break it up according to some fundamental property of methods, such as "physical", "chemical", "biological" (which could create new problems, though). In the short term, I suggest to duplicate name and description if the method is the same but happens to be relevant for observations from the various domains. That way, observations from the various domains can be documented. The notation can actually be the same for the same method. Example: 
- http://codes.wmo.int/wmdr/ObservingMethodTerrestrial/260
- http://codes.wmo.int/wmdr/ObservingMethodAtmosphere/260
Note the slightly different names/labels here! Not good, this should be reconciled. 
@amilan17 Would you agree if I said that we could use this same notation also in a future table ObservingMethod, where we add 'Atmosphere' and 'Terrestrial' as tags?

**Comment by jbianchi81** [comment]
Please check this new proposal, with descriptions added and terms reconciled with [ObservingMethodAtmosphere](http://codes.wmo.int/wmdr/_ObservingMethodAtmosphere) :

[5-02-05 Methods (Terrestrial) proposal.csv](https://github.com/wmo-im/wmds/files/6561041/5-02-05.Methods.Terrestrial.proposal.csv)

I agree with a common table for Methods for all domains in the medium term.

**Comment by jbianchi81** [comment]
I suggest, in addition to the preceding, 4 more methods:
|notation |Domain                                           |Name                                                                                                                                                                           |description           |tags|
|---------|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
|new id   |Terrestrial                                      |Most probable number                                                                                                                                                           |The most probable number (MPN) is a statistical method used to estimate the viable numbers of bacteria in a sample by inoculating broth in 10-fold dilutions and is based on the principle of extinction dilution.|Water quality, Ex situ|
|new id   |Terrestrial                                      |Gram stain                                                                                                                                                                     |Method of staining used to distinguish and classify bacterial species into two large groups: gram-positive bacteria and gram-negative bacteria.                                                                   |Water quality, Ex situ|
|new id   |Terrestrial                                      |Plate  count                                                                                                                                                                   |The plate count method relies on bacteria growing a colony on a nutrient medium so that the colony becomes visible to the naked eye and the number of colonies on a plate can be counted.                         |Water quality, Ex situ|
|new id   |Terrestrial                                      |ATP testing                                                                                                                                                                    |An ATP test is the process of rapidly measuring active microorganisms in water through detection adenosine triphosphate (ATP).  It gives a direct measure of biological concentration and health.                 |Water quality, Ex situ|

**Comment by joergklausen** [comment]
@fstuerzl Please create branch. 
@jbianchi81 Please get a second opinion on accuracy of branch within the next couple days.

**Comment by jbianchi81** [comment]
I updated the proposal on the first comment

**Comment by fstuerzl** [comment]
@jbianchi81, I have created the branch from your proposal (thanks for including the csv!). One question: Can we really replace the method with id 194 by the new term you proposed? I'm not sure, they represent the same concept.

before: 
|notation|path|name|description|
|--|--|--|--|
|194|\Terrestrial\Soil temperature\In situ\Soil thermometer|Soil thermometer||

new:
|notation    |Domain     |Name                                                                                                             |description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |tags                                                                  |
|------------|-----------|-----------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|
|194         |Terrestrial|Liquid-in-glass thermometer (mercury-free)                                                                       |In the LIG thermometer the thermally sensitive element is a liquid contained in a graduated glass envelope. The principle used to measure temperature is that of the apparent thermal expansion of the liquid (http://automationwiki.com/index.php/Liquid-In-Glass_Thermometers, 2021).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |Temperature, water quality, In situ                                   |

I think this change goes beyond typographical edits or additional information (see https://github.com/wmo-im/wmds/wiki/Guidance-for-Removal-of-Terms), so we probably need to create a new entry for "Liquid-in-glass thermometer (mercury-free)" with a new id. 
In OSCAR/Surface the method "Soil thermometer" is frequently used (at 61 stations), so I'm not sure, we can remove the entry 194 without a replacement.

**Comment by fstuerzl** [comment]
Branch: https://github.com/wmo-im/wmds/blob/issue281/tables_en/5-02-05.csv

**Comment by jbianchi81** [comment]
I agree on the change on entry 194 proposed by @fstuerzl . 
By the way, I am curious on how to perform a search by Method on Oscar/Surface...

**Comment by fstuerzl** [comment]
@jbianchi81, thanks for your feedback. Could you also provide a description for entry 194? 

|notation|path|name|description|
|--|--|--|--|
|194|soil temperature, in situ|Soil thermometer||
|new id|temperature, water quality, in situ| Liquid-in-glass thermometer (mercury-free)|In the LIG thermometer the thermally sensitive element is a liquid contained in a graduated glass envelope. The principle used to measure temperature is that of the apparent thermal expansion of the liquid (http://automationwiki.com/index.php/Liquid-In-Glass_Thermometers, 2021).|

**Comment by jbianchi81** [comment]
Soil thermometers can be based on almost any physical principle, so the description has to be quite general:
"A thermometer specifically designed to measure temperature of the soil"

**Comment by fstuerzl** [comment]
@jbianchi81, I have updated the proposal above with this solution. @joergklausen, I think, this can be moved to "validated" now.

**Comment by jbianchi81** [comment]
An expert team of hydrologists has been formed to finalize the proposal. The goal is to deliver a reduced list of non-controversial terms for FT22-1, and then to continue working on the rest.

**Comment by joergklausen** [comment]
@jbianchi81 Thank you! Please make sure to reflect on the following during the process:
1. Does an entry specify a method (=biogeophysicochemical) principle (e.g., tensiometry, UV spectrometry) or a 'conventional' method, named after someone (e.g. Griess Saltzmann)? The answer should be 'yes'.
2. Does a similar entry exist in any of the other 5-02-x tables? If the answer is 'yes' and it is only the domain that is different, then please clearly state this. 
We have recruited Jietze van der Meulen to also help with a holistic review of the methods table(s) with the intention to organize it (even) better. A desired result of this is that we can re-combine the existing methods tables into one, and replace the domains in the current path with tags. Thus, a method like 'electric thermometer' would only appear once in the table with tags 'atmosphere, hydrology, terrestrial, soil, cryosphere, ...'.

**Comment by efucile** [comment]
@joergklausen and @amilan17 we must ensure that we minimise the number of URLs in the description and that we refer to authoritative sources. The reference in a WMO term should be to a WMO publication or other vocabulary. I see http://automationwiki.com/index.php/Liquid-In-Glass_Thermometers that doesn't look like an authoritative source and the link is not working.

**Comment by efucile** [comment]
> I suggest, in addition to the preceding, 4 more methods:
> 
> notation	Domain	Name	description	tags
> new id	Terrestrial	Most probable number	The most probable number (MPN) is a statistical method used to estimate the viable numbers of bacteria in a sample by inoculating broth in 10-fold dilutions and is based on the principle of extinction dilution.	Water quality, Ex situ
> new id	Terrestrial	Gram stain	Method of staining used to distinguish and classify bacterial species into two large groups: gram-positive bacteria and gram-negative bacteria.	Water quality, Ex situ
> new id	Terrestrial	Plate count	The plate count method relies on bacteria growing a colony on a nutrient medium so that the colony becomes visible to the naked eye and the number of colonies on a plate can be counted.	Water quality, Ex situ
> new id	Terrestrial	ATP testing	An ATP test is the process of rapidly measuring active microorganisms in water through detection adenosine triphosphate (ATP). It gives a direct measure of biological concentration and health.	Water quality, Ex situ

"Most probable number" is a generic name with a very wide meaning. It could be used in many contexts different from the intended. Please choose something more specific or delete the definition.

**Comment by efucile** [comment]
The need for new entries needs to be documented. The aim of the WIGOS metadata representation is to support OSCAR/surface in the definition of observing platforms. We need to have a specific need to request for an entry. I believe that we need to have examples of the use of all the proposed terms in observing platforms that are going to be registered in OSCAR. The WIGOS metadata representation is not an academic exercise of classification.

**Comment by efucile** [comment]
126 | Terrestrial | Gravimetric moisture determination | The sample is weighed, oven dried until there is no further mass loss, and then reweighed (https://lter.kbs.msu.edu/protocols/24). | Soil moisture, In situ
-- | -- | -- | -- | --|

is this https://lter.kbs.msu.edu/protocols/24 and authoritative source ?

**Comment by efucile** [comment]
Please delete all the wikipedia references. We cannot refer to Wikipedia from WMO technical regulations. We need to have an authoritative source for the definition.

**Comment by jbianchi81** [comment]
Proposal updated (see first comment)

**Comment by amilan17** [comment]
@jbianchi81 - this looks really well thought out and more ready for FT22-2. It would be helpful to have this one table broken up into 3+ tables for new, revision, deprecation, supersede....

**Comment by jbianchi81** [comment]
[Branch](https://github.com/wmo-im/wmds/blob/issue281/tables_en/5-02-05.csv) updated according to the proposal. First column indicates whether it is a revision, an addition or keeping of an existing term

**Comment by amilan17** [comment]
@jbianchi81 -- we can't use this branch with an added column. Please redo without any columns added. Indication of revision or new should only be in the issue summary at the top of this page.  Please let me know if you have questions about this process.

**Comment by jbianchi81** [comment]
@amilan17 -- I removed the extra column from the branch and added it in the issue summary. Please check if it is correct

**Comment by joergklausen** [comment]
The list is much improved. I am willing to accept it as is, but I still encourage you to scrutinize it once more. Some of the definitions are still a bit verbose and could be shortened (avoiding whole sentences or even several sentences), leaving the reference for those interested in more detail. For example, 

**187**: Determination of electrical resistance between two electrodes embedded in blocks of a porous material buried in the ground and in equilibrium with the surrounding moist soil (WMO Guide to Hydrological Practices, 2008 4.5.3.2 https://library.wmo.int/doc_num.php?explnum_id=10473)

'**Echo sounder**' is a relatively generic name (which is good), but the definition refers specifically to a hydrological measurement. This make it impossible to use a method for anything else, then. Moreover, the name could be improved to reflect the method rather than an instrument. Remember, the instrument 'echo sounder', ideally a concrete make and model, can and should be included in the instrument catalogue. The methods defined here should ideally be describing physical or chemical or biological principles, not individual instrument types, along the lines of '**260**'.

(new)  Echo sounding: Evaluation of the return time of sound waves emitted into a medium to determine distance or depth.

I agree with @fstuerzl that the proposed change to **194** is more than editorial. The old name of 194 'soil thermometer' is not very useful, though. The new name for item 194 is a duplicate of 144 (for which the description is missing). The problem we have at this point is that 144 is found in the 'atmosphere' table only, although the entry is equally relevant for the 'terrestrial' table. The notations are unique (no overlap between these 2 tables, AFAIK). There are 2 approaches to improving this:

1. The tables 5-02-x are combined into a new table 5-02 now. Tags 'atmosphere' and 'terrestrial' are added as appropriate. The description proposed by @jbianchi81 is added to 144, and the existing element 194 is deprecated (or left standing as a generic term).
2. The existing element 194 is deprecated, and a new element with its own notation but same name as 144 is added to the 'terrestrial' list, the proposed definition is also used for 144. The 2 tables are combined at a later stage, and one of the duplicate entries is deprecated then.

I am very much in favor of option 1. @amilan17 Can this be done now for FT2022-2? The problems with duplicates will only become worse the longer we wait.

Methods with '**current**' in their name should be expanded to 'hydrological current' or 'water current' to clearly distinguish from 'electrical current'.

'**Displacement method**' is a bit too generic given the very specific definition.

Should '**Heat flux plate**' be re-named to 'Soil heat flux plate', or are there other applications of this method outside the soil physics domain? Anyway, a definition should be added.

**Comment by jbianchi81** [comment]
@joergklausen -- thanks for your comment. I shared it with my colleagues at JET-HYDMON for feedback. There is an agreement to address the changes proposed. So I will update the proposal and the branch accordingly.

For the record, I share JET-HYDMON feedback:  

#### From Washington Otieno:
I also support the idea of domain specific because our definition are specific too. . Example for displacement method we can make it sediment displacement method, water displacement method,

I agree with soil heat flux plate, because I saw a research from India that you can use the same method for walls but not only soil.

Since will register these variables in OSCAR with already some methods and variables, that are general or specific to give a given application area, then the only way to differentiate different definitions for the same method or variable is to make them specific.

#### From Johanna Korhonen:
I agree also to address changes proposed.
#### From Paolo Leoni:
I agree on the transition from eco sounder to eco sounding.
I suggest to implement @Joergklausen proposal
Echo sounding: Evaluation of the return time of acoustic waves emitted into
a medium to determine the distance between sonar and target.

Speaking of 194 "soil thermometers", I too support the proposed option 1

Speaking current meter we can improve the definition extended it.
"current meter to measure stream velocity" (source:
https://library.wmo.int/doc_num.php?explnum_id=219; p. 75, 5.2 "general
description of...")

Speaking Displacement method, I would suggest adding "of sediments" to
complete the name: "Displacement method of sediments"

**Comment by jbianchi81** [comment]
Proposal and branch updated according to the previous comments.

**Comment by jbianchi81** [comment]
[HCP-4](https://community.wmo.int/activity-areas/hydrology-and-water-resources/meetings/fourth-hydrological-coordination-panel-meeting-hcp-4) has endorsed the proposed code list for variables and methods (12 May 2022). 
In the revision process USGS has proposed the following changes and one addition:

| name | description |
|---------|----------------|
|Pressure-actuated recording gauge | Various ~~pressure-actuated~~ recording gauges in common use operate~~ting~~ on the principle that static pressure at a fixed point in the stream is directly proportional to the head of liquid above the point. This relation is described by the following equation: Water level = (P_static – P_atm ) C, where P_static is the pressure in bar on a fixed spot in the water column, P_atm is the atmospheric pressure in bar on the surface of the water column, and C is a factor of the water’s net weight (C = 10.2 for freshwater at 20°C), which changes with the temperature and salinity of the water.|
|Velocity-area method | Method of determining the discharge of a stream by measuring the <ins>width</ins>, velocity<ins>, and depth</ins> of the flowing water ~~and the depth of the water~~ at a number of ~~points~~ <ins>elemental areas</ins>, over the cross-section, and summing the products ~~of mean velocities by elemental areas~~.|
|Float gauging| Measurement of the velocity of a stream by means of <ins>timing the downstream transit</ins> of a float or velocity rod.|
|Acoustic doppler current profiler (ADCP) | ~~ADCP instruments can be mounted on a moving vessel, such as an inflatable boat. The~~ <ins>An</ins> instrument <ins>that</ins> uses an acoustic doppler transducer to measure water velocity, depth and vessel path simultaneously to compute discharge. As an ADCP instrument processes the signal reflected off the particles in the water, it divides the water column into a number of discrete segments stacked in the vertical. These segments are called depth cells. An ADCP instrument determines the velocity and direction of <ins>flow through</ins> each depth cell. At the same time the <ins>reflected</ins> signal from the bottom, called bottom-track, measures the speed and direction of the boat. Once the profile is finished the ADCP calculates the total discharge <ins>using matrix multiplication</ins>. At least four transects are recommended to calculate the discharge at a site. The actual river discharge estimate will then be the average of the N individual transects discharge values. <ins>ADCP instruments can be mounted on a moving vessel, such as an inflatable boat.</ins>|
|Water discharge-sediment discharge rating curve | ~~A continuous record of suspended-sediment discharge is computed from a record of stream discharges and systematic samples of suspended-sediment concentration. The samples should be taken daily during periods of low and mean flow and more frequently during floods~~<ins>An empirical relationship relating sediment discharge and streamflow based on correlation analysis of sediment samples and concurrent flow estimates and presented as a curve, table, or equation.</ins>|
|Water discharge-bed material rating curve | ~~Measurements of bed-material discharge are made at various stream discharges so that a rating may be prepared showing the relationship between stream discharge and bed-material discharge.~~<ins>An empirical relationship relating bed-material discharge and streamflow based on correlation analysis of sediment samples and concurrent flow estimates and presented as a curve, table, or equation.</ins>|
|<ins>Volumetric method</ins>|<ins>A process for measuring the discharge of a well, spring, or stream by diverting its flow into a volumetric container and timing the fill.</ins>|

**Comment by joergklausen** [comment]
Thank you @jbianchi81 for reporting back from HCP-4. I am happy with most of their suggestions, but I would propose the following additional (purely editorial) changes for the following elements (their descriptions went too far for a vocabulary IMHO):

| name | description |
|---------|----------------|
|Pressure-actuated recording gauge | Recording gauge operating on the principle that static pressure at a fixed point in the stream is directly proportional to the head of liquid above the point, as described by the following equation: Water level = (P_static – P_atm ) C, where P_static is the pressure in bar on a fixed spot in the water column, P_atm is the atmospheric pressure in bar on the surface of the water column. C is a factor of the water’s net weight (C = 10.2 for freshwater at 20°C), which changes with the temperature and salinity of the water.|
|Acoustic doppler current profiler (ADCP) | An instrument that uses an acoustic doppler transducer to measure water velocity, depth and vessel path simultaneously and at various depths to compute discharge. The total discharge is computed from the the profile information using matrix multiplication|

**Comment by jbianchi81** [comment]
Thanks @amilan17 @joergklausen -- I updated the branch removing hlinks from the references. We can discuss the new definitions at the meeting

**Comment by amilan17** [comment]
[https://github.com/wmo-im/wmds/wiki/2022-05-19-TT-WIGOSMD-22](https://github.com/wmo-im/wmds/wiki/2022-05-19-TT-WIGOSMD-22) notes: 

*   @jbianchi81 update branch 
*   Johanna will validate the content of the branch
*   tag Joerg when ready for validation

**Comment by jbianchi81** [comment]
Branch updated: <https://github.com/wmo-im/wmds/commit/4421bfd3e1f9f13f2d177b2265aec3f3df4fd04f>

**Comment by jkorhonen** [comment]
Thank you @jbianchi81 for update!

I made consistency check, minor stylistic things:
159, definition typo ‘measurment’
187,346, 348, 364, 366: Missing comma between definition and reference.

Some descriptions are still very long, maybe formulas/equations could be left out if need to shorten?
Shortening possible 356, 372, 369 last sentence can be left out.

In methods path for water level is "water stage", while in variables it is Water level /Stream level /Lake level. I don't know if it matters.

**Comment by jkorhonen** [comment]
@joergklausen I do not have additional points to raise in addition to stylistic things mentioned above. Consistency check done on my side.

**Comment by amilan17** [comment]
@jbianchi81 @joergklausen (cc @wmo-im/tt-wigosmd) - I'm doing some editorial clean up on this branch and I have a question/concern about citing publications in the definitions. Do you think it's ok to cite a publication if the definition and/or the code name has been modified? 

For example, this is the text in the WMO-No. 385:  

```
gravimetric method
(3) In soil science: method used to measure the amount of water in a soil by weighing the sample before and after drying it in an oven at a temperature of 100–110 degrees Celsius.
```

This is the text in the branch:

```
Gravimetric moisture determination
Measurement of the amount of water in soil by weighing a sample before and after drying it in an oven at a temperature of 100–110 degrees Celsius. International Glossary of Hydrology (WMO-No. 385), 2012 (3rd ed.)
```

  
Ideas alternative options: 

1.  Don't include the reference to the publication, only keep references in the Issue summary.
2.  Preface the publication name with "source:" to indicate that it's an exact copy.
3.  Preface the publication with "reference:" to indicate that it was used and modified.

**Comment by amilan17** [comment]
to discuss:
1. when to cite a publication as a reference in the definition
2. remove formulas from definitions

**Comment by joergklausen** [comment]
IMHO, differentiating between 'source' and 'reference' is splitting hairs and creates rather more confusion. These terms are largely equivalent, and as such, it would be god to use one or the other in our code lists. As a scientist, I would also argue that it is okay to reference a source (sic!) even if we deviate slightly from the exact text in that source but keep the meaning.

**Comment by joergklausen** [comment]
I have updated the original issue, removing all the excessive mathematical detail that is not needed for a general understanding of the principles.

**Comment by amilan17** [comment]
[https://github.com/wmo-im/wmds/wiki/2022-05-31-TT-WIGOSMD-23](https://github.com/wmo-im/wmds/wiki/2022-05-31-TT-WIGOSMD-23) notes: 

*   @amilan17 create template text for references
*   @jbianchi81 update branch based on Joerg's review in issue summary and tag Joerg when done

**Comment by amilan17** [comment]
@jbianchi81 @joergklausen this is my recommendation for including references to pubs in the code definition: 

1.  When the name or definition differs use this text: \[Based on XYZ: _publicationDetails_\]. 
2.  When name and definitions is exact use this text: \[Source: _publicationDetails_\].

**Comment by joergklausen** [comment]
> @jbianchi81 @joergklausen this is my recommendation for including references to pubs in the code definition:
> 
> 1. When the name or definition differs use this text: [Based on XYZ: _publicationDetails_].
> 2. When name and definition is exact use this text: [Source: _publicationDetails_].

I discussed this with @amilan17 and we concluded as follows:
- use [] to denote references/sources, but do not include the word source at the beginning.
- If a description os based on another term, use the syntax: based on term '<term>' in <source>
- If the name of a term has synonyms we want to list, add thes in between the description and the reference using the following format: (synonym: <term>, <term>, ...)

**Comment by jbianchi81** [comment]
@joergklausen -- branch updated

**Comment by amilan17** [comment]
@joergklausen - please validate

**Comment by joergklausen** [comment]
Branch validated, I couldn't see anything wrong ... it's a long list ...

**Comment by amilan17** [comment]
@jbianchi81 there is one code in this table that has a name change, but no definition. Was this intentional or a typo? 

cc @joergklausen 

<table><tbody><tr><td>128</td><td>Soil heat<s>Heat</s> flux plate</td><td>&nbsp;</td></tr></tbody></table>

**Comment by amilan17** [comment]
@jbianchi81 there is one code in this table with a name change, but no definition. Was this inentional or a typo? 

cc @joergklausen 

<table><tbody><tr><td>128</td><td>Soil heat<s>Heat</s> flux plate</td><td>&nbsp;</td></tr></tbody></table>

**Comment by jbianchi81** [comment]
> @jbianchi81 there is one code in this table with a name change, but no definition. Was this inentional or a typo? 
> 
> cc @joergklausen 
> 128	Soil heatHeat flux plate

No definition proposed for this term.

**Comment by amilan17** [comment]
ready for FT
